label.py

```diff
@@ -359,8 +360,29 @@ def show_help():
     print('======================================================================================\n')
 
 
+def images_cmp(x, y):
+    img_list_x = x.split('_')
+    img_list_y = y.split('_')
+    if len(img_list_x) != len(img_list_y):
+        return len(img_list_x) - len(img_list_y)
+    for (f1, f2) in zip(img_list_x, img_list_y):
+        if f1 == f2:
+            continue
+        if f1.isdigit():
+            return int(f1) - int(f2)
+
+
 def main():
+    global images_to_show
     show_help()
+
+    images_to_show = sorted(images_to_show, key=functools.cmp_to_key(images_cmp))
+    bug_ids = set([file_name.split('_')[0] for file_name in images_to_show])
```

Comment:
Another case where it would be useful to have the parsing function in utils :)

Reply:
:smile: Yeah! It feels a needful and will be easier to understand. I will add it after this PR is merged, changing it in both collect.py and here.